package.json

```diff
@@ -2,12 +2,18 @@
   "name": "readable-http",
   "version": "1.0.0",
   "main": "index.js",
+  "targets": {
+    "main": false
+  },
   "author": "James Judd",
   "license": "MIT",
   "devDependencies": {
+    "parcel": "^2.0.0-beta.3.1",
     "typescript": "^4.3.5"
   },
   "scripts": {
-    "build": "mkdir -p dist && cp index.html dist/index.html && tsc"
+    "prebuild": "rm -rf dist",
+    "build": "parcel build ./index.html",
+    "start": "parcel serve ./index.html"
```

Comment:
Parcel by default uses the `./dist` folder both for local dev mode, AND when you run `yarn build` 🤷‍♂️ 

You can configure that if you need.=